doc/source/rllib/package_ref/utils.rst

```diff
@@ -3,16 +3,203 @@
 RLlib Utilities
 ===============
 
-The RLlib ``utils/`` directory contains useful tools and utilities for RL and deep-learning:
-
-.. toctree::
-   :maxdepth: 1
-
-   utils/exploration.rst
-   utils/schedules.rst
-   utils/annotations.rst
-   utils/framework.rst
-   utils/tf_utils.rst
-   utils/torch_utils.rst
-   utils/numpy.rst
-   utils/deprecation.rst
+Here is a list of all the utilities available in RLlib.
+
+Exploration API
+---------------
+
+Exploration is crucial in RL for enabling a learning agent to find new, potentially high-reward states by reaching unexplored areas of the environment.
+
+RLlib has several built-in exploration components that
+the different algorithms use. You can also customize an algorithm's exploration
+behavior by sub-classing the Exploration base class and implementing
+your own logic:
+
+Built-in exploration components
```

Comment:
```suggestion
Built-in Exploration components
```

Reply:
Sorry, I didn't realize that Exploration is an RLlib component. In that case, it should be capitalized.